src/functions/lune-integration/lune.js

```diff
@@ -0,0 +1,85 @@
+const fetch = require("node-fetch");
+const { config } = require("../../../config.js");
+
+/**
+ * Receives the request, gets the lune estimate ID for the selected shipping rate, and creates and order on lune.
+ *
+ * @param {Object} requestEvent the request event built by Netlify Functions
+ * @returns {Promise<{statusCode: number, body: string}>} the response object
+ */
+async function handler(requestEvent) {
+  try {
+    const cartDetails = JSON.parse(requestEvent.body);
+    const ratePrefix = "rate_id_";
+    const selectedShippingRateID =
+      cartDetails["_embeded"]["fx:shipment"].shipping_service_id;
```

Comment:
cartDetails["_embeded"]["fx:shipment"].shipping_service_id; ->  cartDetails["_embedded"]["fx:shipments"][0].shipping_service_id;

There can be multiple shipments for an order, so here I'm just getting the first one. not sure if we want to update the readme to note this limitation in the example